crates/sui-rpc-api/proto/sui/rpc/v2alpha/move_descriptor.proto

```diff
@@ -0,0 +1,210 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package sui.rpc.v2alpha;
+
+// Describes a data type.
+message Descriptor {
+  // Name of this DataType
+  optional string name = 1;
```

Comment:
I noticed the convention here of putting the name of the thing first, and then its package and module -- is that a gRPC convention? I'm used to seeing a type identified by its package, then module, then name. Not a big deal, but I noticed so I thought I'd ask!

Reply:
Order here really doesn't matter too much, happy to reorder. I was just sketching things out and picked a way without too much thought